backend/app/api/endpoints/oidc.py

```
@@ -191,15 +239,21 @@ async def oidc_callback(
 
-        redirect_url = f"{settings.FRONTEND_URL}/login/oidc?access_token={jwt_token}&token_type=bearer&login_success=true"
+        redirect_url = _build_frontend_url(
+            f"/login/oidc?access_token={jwt_token}&token_type=bearer&login_success=true",
+            frontend_base_path,
+        )
         if redirect_after_login:
             redirect_url += f"&redirect={quote(redirect_after_login)}"
 
         return RedirectResponse(url=redirect_url, status_code=302)
 
     except Exception as e:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)